packages/grafana-ui/src/components/Forms/Checkbox.story.tsx

```diff
@@ -12,12 +12,22 @@ export default {
   },
 };
 
-export const simple = () => {
+export const controlled = () => {
   const [checked, setChecked] = useState(false);
   return (
     <Checkbox
       value={checked}
-      onChange={setChecked}
+      onChange={e => setChecked(e.currentTarget.checked)}
```

Comment:
Nit: maybe move anonymous function to `useCallback` to prevent unnecessary rerenders.

Reply:
This is in story maybe, isn't that necessary